core-services/sanitize-prow-jobs/_config.yaml

```diff
@@ -1629,139 +1629,6 @@ groups:
     paths:
     - ".*-postsubmits.yaml$"
   "api.ci":
-    jobs:
-    - periodic-ci-azure-conformance
```

Comment:
I would separate moving jobs with another PR also start to test one job only.

Reply:
We currently don't have any activity in this area. If worry being multiple failures -ok, If interruption - we don't mind this at this point.  Your call.

Reply:
I would separate.
Easy to revert upon issues. :)